index.bs

```diff
@@ -6007,7 +6009,7 @@ ownership will be treated.
 
 [=Remote end definition=]
 
-<pre class="cddl remote-cddl">
+<pre class="cddl remote-cddl local-cddl">
```

Comment:
Also this.

Reply:
It can't be since `script.LocalValue` exists in `local-cddl`.